docs/ReleaseNotes.md

```diff
@@ -33,7 +33,7 @@ This release also updates downstream dependency versions. Most notably, the prot
 * **Feature** Feature 1 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Feature** Feature 2 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Feature** Feature 3 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
-* **Feature** Feature 4 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
+* **Feature** Support READABLE_UNIQUE_PENDING IndexState [(Issue #1611)](https://github.com/FoundationDB/fdb-record-layer/issues/1611)
```

Comment:
It's probably worth noting here or on the issue (I'm not sure what the best practice is), that using this new state is not compatible with older code.

Reply:
Should it be a breaking change?

Reply:
Should the warning be in the release notes or in the `allowUniquePending` javadoc?